samples/servicecontrol/adapter-rabbitmq-different-topologies/sample.md

```diff
@@ -33,12 +33,12 @@ NOTE: In order to connect to a different RabbitMQ broker, ensure all connection
 
 ## Running the project
 
- 1. Start the projects: Adapter, Sales and Shipping (right-click on the project, select the `Debug > Start new instance` option). Make sure adapter starts first because on start-up it creates a queue that is used for heartbeats.
+ 1. Start the projects: Adapter, Sales and Shipping (right-click on the project, select the `Debug > Start new instance` option). Make sure the adapter starts first because on start-up it creates a queue that is used for heartbeats.
```

Comment:
> Start the projects: Adapter, Sales and Shipping (right-click on the project, select the `Debug > Start new instance` option).

this doesnt really make sense. if we want them to runn all project why confuse them with "right click". if they have downloaded the sln from docs all those will be default startup projects and we should just be able to tell them to "run the solution"